src/sentry/dynamic_sampling/tasks/performance_timing.py

```diff
@@ -0,0 +1,93 @@
+from sentry import options
+from sentry.dynamic_sampling import get_redis_client_for_ds
+from sentry.dynamic_sampling.tasks.common import get_active_orgs_with_projects_counts
+from sentry.dynamic_sampling.tasks.utils import Timer
+from sentry.tasks.base import instrumented_task
+
+PERFORMANCE_TIMING_LOCK_NAME = "ds:performance_timing"
+
+
+@instrumented_task(
+    name="sentry.dynamic_sampling.tasks.performance_timing",
+    queue="dynamicsampling",
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=2 * 60 * 60,
+    time_limit=2 * 60 * 60 + 5,
+)
+def performance_timing() -> None:
+    enabled = options.get("dynamic-sampling.timing.task") or False
+
+    if not enabled:
+        return
+
+    if not task_lock(PERFORMANCE_TIMING_LOCK_NAME, 60):
+        # the task is already running don't run it again
+        return
+
+    try:
+        time_active_orgs_with_project_counts()
+
+    finally:
+        task_unlock(PERFORMANCE_TIMING_LOCK_NAME)
+
+
+def time_active_orgs_with_project_counts():
+    """
+    Measures the time of get_active_orgs_with_project_counts()
+    :return:
+    """
+    enabled = options.get("dynamic-sampling.timing.get_active_orgs_with_project_counts")
+
+    if not enabled:
+        return
+
+    num_orgs = options.get("dynamic-sampling.timing.get_active_orgs_with_project_counts.num_orgs")
+
+    t = Timer("get_active_orgs_with_projects_counts")
+    t.start()
+    org_count = 0
+    proj_count = 0
+    for orgs in get_active_orgs_with_projects_counts(num_orgs):
+        org_count += len(orgs)
+        for org_projects in orgs:
+            proj_count += org_projects
```

Comment:
return signature of `get_active_orgs_with_projects_counts` is incorrect. Number of projects is returned as a seecond element of a tuple